source/Gesture Recognizer/Assets/Plugins/GestureRecognizer/Demo/Scripts/DetectGesture.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace GestureRecognizer.Demo
+{
+    public class DetectGesture : MonoBehaviour
+    {
+        #region FIELDS
+
+        private const string NoDatasetFoundMessage = "No gesture dataset";
+        private const string NoGesturesFoundMessage = "No gestures found";
+        private const string GestureDetectedMessage = "{0}  recognized!";
+
+        [SerializeField] private GesturesDataset gestureDataset = null;
```

Comment:
according to this https://bit.ly/30A1m8v private fields come before Serialized fields

Reply:
The convention should be for each group of elements to be ordered from more accessible to more private, we looked up the official rule of StyleCop and the conventions file was changed accordingly http://stylecop.soyuz5.com/SA1202.html